src/features/rewards/tableContribute/index.tsx

```diff
@@ -144,16 +144,14 @@ export default class TableContribute extends React.PureComponent<Props, {}> {
       if (this.props.showRowAmount) {
         if (this.props.showRemove) {
           const remaining = (100 - row.attention) / 1.04
-          const attention = row.attention / 1.04
-          const diff = remaining + attention
           cell.customStyle = {
             background: `linear-gradient(
               to right,
               transparent 0%,
               transparent ${remaining}%,
               rgba(210, 198, 243, 0.39) ${remaining}%,
-              rgba(210, 198, 243, 0.39) ${diff}%,
-              transparent ${diff}%,
+              rgba(210, 198, 243, 0.39) 100%,
```

Comment:
Using anything other than 100% was causing an unnecessary right padding in the fill up bar. Now it hugs the right side

Reply:
@ryanml we shouldn't do that, because we have remove button on the right side which needs to have white bg.

![image](https://user-images.githubusercontent.com/9574457/47036998-0ed93300-d17e-11e8-9c2f-a876487c562c.png)

Reply:
problem now

![image](https://user-images.githubusercontent.com/9574457/47037806-3cbf7700-d180-11e8-926a-e4a67cc6e5ca.png)

Reply:
Can the background and content be contained in it's own element so that it does not have to worry about what its inside or what else is inside the parent?

Reply:
@petemill problem here is that we have table and we add background to the tr

Reply:
Ah, tables.... 😬 Careful!

Reply:
Issue here guys (at least on mobile) is that the above is the expected result, see:

<img width="399" alt="screen shot 2018-10-16 at 12 53 25 pm" src="https://user-images.githubusercontent.com/8732757/47043474-930dc500-d142-11e8-914b-dc7f425eed60.png">

I definitely don't want to make a new component just on this account, any suggestions?